dot/telemetry/telemetry.go

```diff
@@ -57,126 +54,81 @@ var (
 )
 
 // GetInstance singleton pattern to for accessing TelemetryHandler
-func GetInstance() *Handler {
+func GetInstance() *Handler { //nolint
 	if handlerInstance == nil {
 		once.Do(
 			func() {
 				handlerInstance = &Handler{
-					buf: bytes.Buffer{},
+					msg: make(chan Message, 10),
```

Comment:
Increase the buffer size to `256`. 
It may slow down the node if the buffer is full since `SendMessage` is called form `handleBlock`

Reply:
updateded to 256.